lib/fetch-yui-docs.js

```diff
@@ -66,8 +66,8 @@ export default async function fetchYuiDocs(
 		ignorePreviouslyIndexedDoc
 	)
 
-	return [
-		...(await fetchYuiDocsFromNpm(projects, 'ember', filterValidVersionsForRequestedDocs)),
-		...(await fetchYuiDocsFromNpm(projects, 'ember-data', filterValidVersionsForRequestedDocs)),
-	]
+	let emberYuidocs = await fetchYuiDocsFromNpm(projects, 'ember', filterValidVersionsForRequestedDocs)
```

Comment:
Curious, wasn't the previous statement working?

Reply:
It would fail when running local generation because one or the other would be undefined.

Reply:
In that case could you add a comment there so that none of us(mostly me😅) doesn't end up changing that in the future

Reply:
sure

Reply:
done.